### PR TITLE
Prettierify code.

### PR DESCRIPTION
**Pull request by n1k0**
[Prettier](https://github.com/prettier/prettier) is pretty. Let's use it.

### CODE REVIEW

File: test/index_test.js [hunk 1]
```diff
@@ -166,10 +173,10 @@ describe("Kinto", () => {
     });
 
     it("should set collection's remoteTransformers", () => {
-      const transformer = {encode(){}, decode(){}};
+      const transformer = { encode() {}, decode() {} };
       const db = new Kinto();
       const options = {
-        remoteTransformers: [ transformer ]
+        remoteTransformers: [transformer]
```
Comment (glasserc) [inline test/index_test.js]: Weird that objects get spaces inside their braces, but arrays don't..

**Comment by n1k0** [comment]
Test failure is related to https://github.com/Kinto/kinto-node-test-server/pull/1#issuecomment-285797047

**Comment by glasserc** [comment]
I'm hoping to fix it in https://github.com/Kinto/kinto.js/pull/660.

**Comment by n1k0** [comment]
I've enabled es5 trailing commas and installed a git precommit hook to have code prettified automatically, so people don't have to mess with their text editor too much.

**Comment by n1k0** [comment]
I'm also gonna fix the Travis build configuration in this PR.

**Comment by n1k0** [comment]
Green.

**Pull request merged**